lifecycle_msgs/msg/TransitionEvent.msg

```diff
@@ -1,5 +1,5 @@
 # The time point at which this event occurred.
-uint64 timestamp
+builtin_interfaces/Time stamp
```

Comment:
If we do make this change, why change the field name? I would leave it as `timestamp`, as that is more descriptive anyway. I know `stamp` is used in other places (like in the std_msgs/Header), but that is mostly to match ROS 1.